src/Kunstmaan/AdminListBundle/Resources/views/Default/add_or_edit.html.twig

```diff
@@ -46,7 +46,11 @@
         <!-- Fields -->
         <fieldset class="form__fieldset--padded">
             {% block form_content %}
-                {{ form_widget(form) }}
+                {% if tabPane is defined %}
+                    {{ tabs_widget(tabPane) }}
+                {% else %}
```

Comment:
I don't think the else is needed here, but we should always have a form_end(form)